[Bug]: v7 - unable to import react-router/testing

### What version of React Router are you using?

7.0.0-pre.2

### Steps to Reproduce

1. Install `react-router` package in version `7.0.0-pre.2`. 
2. Try to import `import { createStub } from 'react-router/testing'`
3. It will fail, since there is no `testing` export in the `package.json` defined



### Expected Behavior

According to the [dev testing documentation](https://reactrouter.com/dev/guides/start/testing), it should be possible to import from this package.

### Actual Behavior

The build fails with 

```
error TS2307: Cannot find module 'react-router/testing' or its corresponding type declarations.
import { createStub } from 'react-router/testing'
```

--- a/docs/start/testing.md
+++ b/docs/start/testing.md
@@ -5,7 +5,7 @@ order: 9
 
 # Testing
 
-When components use things like `useLoaderData`, `<Link>`, etc, they are required to be rendered in context of a React Router app. The `createStub` function creates that context to test components in isolation.
+When components use things like `useLoaderData`, `<Link>`, etc, they are required to be rendered in context of a React Router app. The `createRoutesStub` function creates that context to test components in isolation.
 
 Consider a login form component that relies on `useActionData`
 
@@ -32,10 +32,11 @@ export function LoginForm() {
 }
 ```
 
-We can test this component with `createStub`. It takes an array of objects that resemble route modules with loaders, actions, and components.
+We can test this component with `createRoutesStub`. It takes an array of objects that resemble route modules with loaders, actions, and components.
 
 ```tsx
-import { createStub, route } from "react-router/testing";
+import { route } from "@react-router/dev/routes";
+import { createRoutesStub } from "react-router";
 import * as Test from "@testing-library/react";
 import { LoginForm } from "./LoginForm";
 
@@ -43,7 +44,7 @@ test("LoginForm renders error messages", async () => {
   const USER_MESSAGE = "Username is required";
   const PASSWORD_MESSAGE = "Password is required";
 
-  const Stub = createStub([
+  const Stub = createRoutesStub([
     route("/login", {
       Component: LoginForm,
       action() {

--- a/docs/start/testing.md
+++ b/docs/start/testing.md
@@ -35,7 +35,6 @@ export function LoginForm() {
 We can test this component with `createRoutesStub`. It takes an array of objects that resemble route modules with loaders, actions, and components.
 
 ```tsx
-import { route } from "@react-router/dev/routes";
 import { createRoutesStub } from "react-router";
 import * as Test from "@testing-library/react";
 import { LoginForm } from "./LoginForm";
@@ -45,7 +44,8 @@ test("LoginForm renders error messages", async () => {
   const PASSWORD_MESSAGE = "Password is required";
 
   const Stub = createRoutesStub([
-    route("/login", {
+    {
+      path: "/login",
       Component: LoginForm,
       action() {
         return {

